packages/solo/src/chain-cosmos-sdk.js

```diff
@@ -332,6 +332,10 @@ ${chainID} chain does not yet know of address ${clientAddr}${adviseEgress(
         };
         // Send that message, and wait for the subscription.
         ws.send(JSON.stringify(obj));
+
+        // Ensure our sender wakes up again.
+        // eslint-disable-next-line no-use-before-define
+        sendUpdater.updateState(true);
```

Comment:
Ok so if I understand correctly, this means that every time we establish a new websocket, we'll try to re-send queued messages, whereas previously we were probably only doing that when a message was first queued, which would probably result in a stall if/when the RPC node got bounced.

Reply:
Exactly!